lua5_3/lua5_3.l

```diff
@@ -22,42 +22,42 @@ true "TRUE"
 until "UNTIL"
 while "WHILE"
 [a-zA-Z_][a-zA-Z_0-9]* "NAME"
-\+ "+"
-\- "-"
-\* "*"
-/ "/"
-% "%"
-\^ "^"
-# "#"
-& "&"
-~ "~"
-\| "|"
-<< "<<"
->> ">>"
-// "//"
-== "=="
-~= "~="
-<= "<="
->= ">="
-< "<"
-> ">"
-= "="
-\( "("
-\) ")"
-\{ "{"
-\} "}"
-\[ "["
-\] "]"
-:: "::"
-; ";"
-: ":"
-, ","
+\+ "PLUS"
+\- "MINUS"
+\* "ASTERIX"
+/ "FSLASH"
+% "MOD"
+\^ "CARET"
+# "HASH"
+& "AMP"
+~ "TILDE"
+\| "PIPE"
+<< "LTLT"
+>> "GTGT"
+// "FSFS"
+== "EQEQ"
+~= "NOTEQ"
+<= "LE"
+>= "GE"
+< "LT"
+> "GT"
+= "EQ"
+\( "ORB"
+\) "CRB"
+\{ "OCB"
+\} "CCB"
+\[ "OSB"
+\] "CSB"
+:: "COLCOL"
+; "SEMICOL"
+: "COL"
+, "COMMA"
 ("(\\[abfnrtvz\\"']|\\x[0-9a-fA-F]|\\[0-9][0-9]?[0-9]?|\\u\{[0-9a-fA-F]+\}|.)*?")|('(\\[abfnrtvz\\"']|\\x[0-9a-fA-F]|\\[0-9][0-9]?[0-9]?|\\u\{[0-9a-fA-F]+\}|.)*?') "SHORT_STR"
 (\[\[.*?\]\])|(\[=\[.*?\]=\])|(\[==\[.*?\]==\]) "LONG_STR"
 (0[xX][0-9a-fA-F]+(\.[0-9a-fA-F]+)?([pP]-?[0-9a-fA-F]+)?)|(\.[0-9]+([eE]-?[0-9]+)?)|([0-9]+(\.[0-9]*)?([eE]-?[0-9]+)?) "NUMERAL"
 (--\[=====\[.*?\]=====\])|(--\[====\[.*?\]====\])|(--\[===\[.*?\]===\])|(--\[==\[.*?\]==\])|(--\[=\[.*?\]=\])|(--\[\[.*?\]\])|(--.*?$) ;
 [ \n\r\t]+ ;
 #!.*?$ ;
-\. "."
-\.\. ".."
-\.\.\. "..."
+\. "D"
```

Comment:
`DOT`?

Reply:
Should `..` be renamed to `DOTDOT` or is `DD` fine?

Reply:
`DOTDOT` sounds right to me.